command.go

```diff
@@ -756,6 +759,25 @@ func (c *Command) ValidateArgs(args []string) error {
 	return c.Args(c, args)
 }
 
+func (c *Command) ValidateRequiredFlags() error {
+	flags := c.Flags()
+	missingFlagNames := []string{}
+	flags.VisitAll(func(pflag *flag.Flag) {
+		requiredAnnotation, found := pflag.Annotations[BashCompOneRequiredFlag]
+		if !found {
+			return
+		}
+		if (requiredAnnotation[0] == "true") && !pflag.Changed {
+			missingFlagNames = append(missingFlagNames, pflag.Name)
+		}
+	})
+
+	if len(missingFlagNames) > 0 {
+		return fmt.Errorf("Required flags \"%s\" have/has not been set", strings.Join(missingFlagNames, "\", \""))
```

Comment:
Or just write `return fmt.Errorf(`Required flags "%s" have/has not been set`, strings.Join(missingFlagNames, `", "`))`. Backslashes look ugly :(